cms/urls.py

```diff
@@ -82,7 +82,10 @@
         debug_urlpatterns = [path("__debug__/", include(debug_toolbar.urls)), *debug_urlpatterns]
 
 # Public URLs that are meant to be cached.
-urlpatterns: list[Union["URLResolver", "URLPattern"]] = []
+urlpatterns: list[Union["URLResolver", "URLPattern"]] = [
+    path("", include("cms.search.urls")),
+]
```

Comment:
Currently, it 404s if it is not an external environment. We should change the behaviour so the route is only registered if it is external, so the endpoint doesn't even get registered in internal.